src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,27 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @ManyToMany(fetch = FetchType.EAGER)
+    @JoinTable(name = "countries_actors",
+            joinColumns = @JoinColumn(name = "country_id"),
```

Comment:
you are joining filelds and tables for movies and actors, not countries and actors

Reply:
My bad)